.drone.yml

```diff
@@ -92,6 +96,13 @@ workspace:
   base: /go
   path: src/code.gitea.io/gitea
 
+trigger:
```

Comment:
Triggers are AND in Drone, so this won't execute for backports

Reply:
you mean backport PRs?

Reply:
Yes, the condition here is that branch must be `master` and event must be either `pull_request` or `push`.

Reply:
yeah, you are right but somehow I can't think of better way without duplicating steps :(